lib/galaxy/dependencies/pinned-requirements.txt

```diff
@@ -91,4 +91,3 @@ python-genomespaceclient==0.1.8
 
 # ISA
 git+https://github.com/ISA-tools/isa-rwval.git@develop#egg=isa-rwval
```

Comment:
Actually can now revert this line back to the package name.

`isa-rwval==0.10.1` should now allow some table parsing to reintroduce the data files displaying.